docker/convert/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM paddlepaddle/paddle
+ADD ./convert.py /convert/
+ADD ./logging.conf /convert/
+ADD .cache/paddle/dataset /root/.cache/paddle/dataset
```

Comment:
Didn't get your point. Default command could be `CMD ["/program", "argument"]`, this can be overrided by k8s yaml definations